### ISSUE TITLE
Fix flaky kudu tests: `schema/table not found` and enable CI

### ISSUE THREAD

**Issue by grantatspothero** [trinodb/trino#11203]
Kudu tests are flaky, see example flake here:
https://github.com/trinodb/trino/runs/5303491394?check_suite_focus=true

```
Error:  io.trino.plugin.kudu.KuduSmokeTests$TestKuduWithDisabledInferSchemaConnectorTest.init  Time elapsed: 55.908 s  <<< FAILURE!
java.lang.RuntimeException: Table 'default.customer' not found
	at io.trino.testing.AbstractTestingTrinoClient.execute(AbstractTestingTrinoClient.java:122)
	at io.trino.testing.DistributedQueryRunner.execute(DistributedQueryRunner.java:505)
	at io.trino.testing.QueryAssertions.copyTable(QueryAssertions.java:339)
	at io.trino.testing.QueryAssertions.copyTable(QueryAssertions.java:331)
	at io.trino.testing.QueryAssertions.copyTpchTables(QueryAssertions.java:323)
	at io.trino.plugin.kudu.KuduQueryRunnerFactory.createKuduQueryRunnerTpch(KuduQueryRunnerFactory.java:107)
	at io.trino.plugin.kudu.KuduQueryRunnerFactory.createKuduQueryRunnerTpch(KuduQueryRunnerFactory.java:83)
	at io.trino.plugin.kudu.AbstractKuduConnectorTest.createQueryRunner(AbstractKuduConnectorTest.java:54)
```

**Comment by grantatspothero** [comment]
PR to address kudu flakes here: https://github.com/trinodb/trino/pull/11185

**Comment by grantatspothero** [comment]
And a followup PR is here: https://github.com/trinodb/trino/pull/11264

**Comment by ebyhr** [comment]
@grantatspothero I believe Kudu test became flaky due to recent changes. Do you know what is the trigger? BaseConnectorTest migration?

**Comment by ebyhr** [comment]
Labeled `RELEASE-BLOCKER` since we disabled all Kudu tests in #11313

**Comment by hashhar** [comment]
Fixed by #11264
